services/core-web/package.json

```diff
@@ -11,6 +11,22 @@
     "@babel/runtime-corejs3": "^7.5.5",
     "@datapunt/matomo-tracker-react": "^0.1.5",
     "@hot-loader/react-dom": "^16.13.0",
+    "@syncfusion/ej2-base": "^19.1.57",
```

Comment:
were these caused by a formatter in vscode? looks like the versions didn't change

Reply:
Yes, it was formatted in that order after upgrading syncfusion/ej2-react-pdfviewer from 19.1.57 to 19.4.55. That's the only change. The reason to update was to get FormDesigner object and be able to inject it.